packages/elements/src/components/ino-checkbox/ino-checkbox.scss

```diff
@@ -16,38 +16,39 @@
    * @prop --ino-checkbox-checkmark-color-disabled: color of the check mark if disabled
    */
 
-  --checkbox-color: var(--ino-checkbox-color, #{theme.color(primary)});
-  --checkbox-color-hover: var(
-    --ino-checkbox-color-hover,
-    #{theme.color(primary, light)}
-  );
-  --checkbox-color-active: var(
-    --ino-checkbox-color-active,
-    #{theme.color(primary, dark)}
-  );
-  --checkbox-container-color-unchecked: var(
-    --ino-checkbox-container-color-unchecked,
-    #{theme.color(primary, light)}
-  );
-  --checkbox-text-color-disabled: var(
-    --ino-checkbox-text-color-disabled,
-    #{theme.color(dark)}
-  );
-  --checkbox-background-color-disabled: var(
-    --ino-checkbox-background-color-disabled,
-    #{theme.color(light, light)}
-  );
-  --checkbox-checkmark-color: var(--ino-checkbox-checkmark-color, #fff);
-  --checkbox-checkmark-color-disabled: var(
-    --ino-checkbox-checkmark-color-disabled,
-    #{theme.color(light)}
-  );
+   --checkbox-color: var(--ino-checkbox-color, #{theme.color(primary)});
+   --checkbox-color-hover: var(
+     --ino-checkbox-color-hover,
+     #{theme.color(primary, light)}
+   );
+   --checkbox-color-active: var(
+     --ino-checkbox-color-active,
+     #{theme.color(primary, dark)}
+   );
+   --checkbox-container-color-unchecked: var(
+     --ino-checkbox-container-color-unchecked,
+     #{theme.color(primary, light)}
+   );
+   --checkbox-text-color-disabled: var(
+     --ino-checkbox-text-color-disabled,
+     #{theme.color(dark)}
+   );
+   --checkbox-background-color-disabled: var(
+     --ino-checkbox-background-color-disabled,
+     #{theme.color(light, light)}
+   );
+   --checkbox-checkmark-color: var(--ino-checkbox-checkmark-color, #fff);
+   --checkbox-checkmark-color-disabled: var(
+     --ino-checkbox-checkmark-color-disabled,
+     #{theme.color(light)}
+   );
 }
 
+
 :host {
-  --mdc-checkbox-unchecked-color: var(--checkbox-container-color-unchecked);
-  --mdc-checkbox-checked-color: var(--checkbox-color);
-  --mdc-checkbox-disabled-color: var(--checkbox-background-color-disabled);
+  --mdc-checkbox-unchecked-color: #ABB1CC;
+  --mdc-checkbox-checked-color: #2C02FF;
+  --mdc-checkbox-disabled-color: #CBD0E0;
```

Comment:
Instead of using these variables, can't we use [these mixins](https://material.io/components/checkboxes/web#api)? I would prefer mixins because that's the "official" API